machine/v1beta1/types_vsphereprovider.go

```diff
@@ -51,6 +51,11 @@ type VSphereMachineProviderSpec struct {
 	// This parameter will be ignored if 'LinkedClone' CloneMode is set.
 	// +optional
 	DiskGiB int32 `json:"diskGiB,omitempty"`
+	// TagIDs is an optional set of tags to add to an instance. Specified tagIDs
+	// must use URN-notation instead of display names.
+	// For example: urn:vmomi:InventoryServiceTag:5736bf56-49f5-4667-b38c-b97e09dc9578:GLOBAL
+	// +optional
+	TagIDs []string `json:"tagIDs,omitempty"`
```

Comment:
How is this handled in the upstream CAPV?

Reply:
This mirrors the CAPV API.